code/ship/ship.cpp

```diff
@@ -657,6 +658,7 @@ ship_flag_description Ship_flag_descriptions[] = {
 	{ Ship_Flags::No_builtin_messages,			"Ship will not send any persona messages."},
 	{ Ship_Flags::Scramble_messages,			"All messages sent from this ship will appear scrambled, as if the ship had been hit by an EMP."},
 	{ Ship_Flags::Maneuver_despite_engines,		"Ship can maneuver even if its engines are disabled or disrupted" },
+	{ Ship_Flags::No_scanned_cargo,             "Ship cargo will never be revealed and will instead only show scanned or not scanned."},
```

Comment:
AFAICT, this flag only works in the new scanning mode. This description should ideally note this.

Reply:
Done. The PR is now also showing the updates Naomi requested.

Reply:
Er.. maybe not. I'm rather confused on my commit history here and it's been long enough I can't recall exactly what Naomi and I landed on for those requests. I wouldn't have said "done" if I didn't do anything.